toolkit/imageconfigs/full-aarch64.json

```diff
@@ -16,8 +20,12 @@
             "Name": "CBL-Mariner Core",
             "PackageLists": [
                 "packagelists/hyperv-packages.json",
-                "packagelists/core-packages-image-aarch64.json"
+                "packagelists/core-packages-image-aarch64.json",
+                "packagelists/selinux.json"
```

Comment:
Let's not include selinux.json or the SELinux KernelCommandLine for the "CBL-Mariner Core" option to keep it in line with the changes from #2313.